packages/blocks/plus_minus/README.md

```diff
@@ -0,0 +1,87 @@
+
+# @blockly/plus-minus [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
```

Comment:
The naming convention we're going with is: @blockly/(field|block|plugin)-NAME.
Since this is a block package, that would be: 
``@blockly/block-plus-minus``